bot/admin/server/src/main/kotlin/service/MetricService.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2017/2022 e-voyageurs technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.tock.bot.admin.service
+
+import ai.tock.bot.admin.indicators.metric.CustomMetric
+import ai.tock.bot.admin.indicators.metric.MetricFilter
+import ai.tock.bot.admin.indicators.metric.MetricGroupBy
+import ai.tock.bot.admin.indicators.metric.MetricDAO
+import ai.tock.bot.admin.model.indicator.metric.toMetricResponse
+import ai.tock.shared.injector
+import com.github.salomonbrys.kodein.instance
+import model.indicator.metric.CustomMetricResponse
+
+object MetricService {
+
+    private val metricDao: MetricDAO by injector.instance()
```

Comment:
```suggestion
    private val metricDao: MetricDAO get() = injector.provide()
```
Pour ne pas avoir les soucis d'injection éventuels rencontrés dans les scénarios côté tests